### ISSUE TITLE
`DocService` returns 404 over non-TLS connection on Chrome

### ISSUE THREAD

**Issue by ks-yim** [line/armeria#4728]
Chrome intentionally advertises Brotli(`br`) as `accept-encoding` header only over a TLS connection to avoid the data being spoiled by non-compliant HTTP proxies.

As a result of this behavior, `DocService` can't be opened on Chrome as Armeria is serving pre-compressed assets in Brotli format for `DocService` but and the backing `FileService` searches for `/index.html` or `/index.html.gz` but not `/index.html.br`.

A potential workarounds for this would be to make `FileService` to return uncompressed content when `accept-encoding` header does not contain the encoding used for pre-compiled assets, but this will force `Brotli native lib` dependency to armeria core, which is not desirable.

Using gzip for pre-compiled static assets appears to be the right way.

Here's curl to reproduce the issue..:
```shell
# Correctly returns 200
curl "http://localhost:8080/internal/docs" -H "accept-encoding: br"
curl "http://localhost:8080/internal/docs" -H "accept-encoding: gzip, br"

# Returns 404 without brotli native lib dependency in the classpath
curl "http://localhost:8080/internal/docs" 

# Returns 404 always even if brotli native lib is in the classpath"
curl "http://localhost:8080/internal/docs" -H "accept-encoding: gzip"
```

**Comment by ks-yim** [comment]
Ah, the fix was already shipped in 1.22.0..! Nice, I will check it out and close the issue 😆.

**Comment by ks-yim** [comment]
Confirmed that `DocService` works well on `1.22.0`, thanks!
